app/src/main/java/com/orange/ods/app/ui/components/buttons/icons/ButtonsIcon.kt

```diff
@@ -67,19 +69,38 @@ fun ButtonsIcon(customizationState: ButtonIconCustomizationState) {
                 )
             }
 
-            CodeImplementationColumn(modifier = Modifier.padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.screen_horizontal_margin))) {
+            CodeImplementationColumn(
+                modifier = Modifier.padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.screen_horizontal_margin)),
+                composeContent = {
                 FunctionCallCode(
-                    name = OdsComposable.OdsIconButton.name,
-                    exhaustiveParameters = false,
-                    parameters = {
-                        classInstance<OdsIconButton.Icon>("icon") {
-                            painter()
-                            contentDescription("")
+                        name = OdsComposable.OdsIconButton.name,
+                        exhaustiveParameters = false,
+                        parameters = {
+                            classInstance<OdsIconButton.Icon>("icon") {
+                                painter()
+                                contentDescription("")
+                            }
+                            if (!isEnabled) enabled(false)
                         }
-                        if (!isEnabled) enabled(false)
+                    )
+                },
+                xmlContent = {
+                    CodeBackgroundColumn {
+                        XmlViewTag(
+                            clazz = com.orange.ods.xml.component.button.OdsIconButton::class.java,
+                            xmlAttributes = {
+                                id("ods_icon_button")
+                                layoutWidth()
+                                layoutHeight()
+                                drawableAppAttr("icon", "icon")
+                                appAttr("iconContentDescription", "Icon description")
+                                if (!isEnabled) disabledAppAttr()
+                                appAttr("odsDisplaySurface", "light")
```

Comment:
I am not sure we should display this attribute because we don't display it for `OdsIconToggleButton`.

However if we display this attribute, its name should be `displaySurface` instead of `odsDisplaySurface`.

Reply:
Yes, I agree, I forgot to remove it.